sycl/test-e2e/Graph/invalid_profiling.cpp

```diff
@@ -0,0 +1,160 @@
+// REQUIRES: level_zero || cuda, gpu
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out 2>&1
+// RUN: %if ext_oneapi_level_zero %{env UR_L0_LEAKS_DEBUG=1 %{run} %t.out 2>&1 | FileCheck --implicit-check-not=LEAK %s %}
+
+// This test checks exception throwing if profiling an event returned
+// from multi-partitions graph submission.
+
+#define GRAPH_E2E_EXPLICIT
```

Comment:
should these new tests live in the "Explicit" sub-directory?

Reply:
These tests have been moved to the "Explicity" sub-directory. However, we need the PR https://github.com/intel/llvm/pull/12366 to be merged to implement "RecordAndReplay" equivalent tests.